libs/movie/src/lib/movie/dashboard/shell/shell.component.ts

```diff
@@ -39,6 +39,7 @@ export class DashboardTitleShellComponent implements OnInit, OnDestroy {
   public movie = firstValueFrom(this.movie$);
 
   @Input() routes: RouteDescription[];
+  @Input() editRoute?: string | any[]; // same type as expected by [routerLink]
```

Comment:
taken from existing component: 
```suggestion
  @Input() editLink: string | string[];
```